MIP/mip-84/README.md

```diff
@@ -0,0 +1,182 @@
+# MIP-84: Lock-Mint Native Bridge: Basics of Relayer trust assumptions
+
+- **Description**: This MIP clarifies the trust assumptions for the Relayer, and what consequences should be drawn from these.
+- **Authors**: Andreas Penzkofer
+- **Desiderata**: [MD-74](../MIP/mip-74)
+
+## Abstract
+
+The lock/mint-type bridge has the following core actors and components: a user who initiates a transfer, a bridge operator, bridge contracts that facilitate the transfer, and a relayer that submits the completion of the transfer to the bridge contracts.
+
+The trust assumptions on the relayer component have significant implications for the security of the bridge and they can introduce the need for additional components.
+
+In a scenario based approach we clarify minimally required components and why they are needed. We distinguish between a trusted, partially trusted, and untrusted relayer (with proofs).
+
+## Motivation
+
+We need to clarify the trust assumptions for the Relayer and what consequences should be drawn from these. In many cases a rate limitation may be necessary, see also the Appendix [A1: Eigenlayer AVS](#a1-background-eigenlayer).
+
+This addresses in [MD-74](https://github.com/movementlabsxyz/MIP/blob/mip/rate-limiter-lock-mint-bridge/MD/md-74/README.md#D1) the following:
+
+- D1 : Specify the actors and their trust assumptions
+- D2 : Specify the risks and threats from components
+
+## Specification
+
+_The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174._
+
+We define the following terms:
```

Comment:
If you think about it, there is not much difference between partially trusted and untrusted.
Any behaviour an untrusted relayer can exhinit can be simulated by a partially trusted relayer with a corresponding "bug".
In DS, we usually don't distinguish between intentional and unintentional  erroneous components because the visible behaviours are the same.

Reply:
Yes, but the response is different. For an untrusted component we do the following

> Any action should be approved by a trusted party and require a proof of correctness.

- in case of ZK we require a ZK proof before applying
- in case of optimistic we wait for 7 days before applying
- in case of FFS we require a committee signature

This does not hold true for partially trusted. There we apply the transaction without any way of reverting.

Reply:
OK, I see. However I am not sure we should make a difference between faulty (bug) or compromised (taken control of) when wee compensate users.

Reply:
that is correct. i fixed this in the untrusted case now